contest/views.py

```diff
@@ -384,3 +387,42 @@ def download(request):
             return render_index(request,'contest/download.html',{'contest':contest})
         else:
             raise PermissionDenied
+
+@login_required
+def rejudge(request):
```

Comment:
It's required to check permission in this function.
Only contest owner / coowner can rejudge contest problem.